.github/workflows/ci.yaml

```diff
@@ -1844,6 +1835,36 @@ jobs:
           ignore-unfixed: true
           vuln-type: 'os,library'
           severity: 'CRITICAL,HIGH'
+  scan-cactus-connector-quorum:
```

Comment:
@petermetz our CI will be flooded with lots of steps with 3 basic actions performed per plugin in general: building, testing, security-check.
Shall we not club together all of these within `per-plugin` job, which does all the tasks per plugin together?

![image](https://github.com/hyperledger/cacti/assets/56861721/39cb2609-aacf-48fc-8c46-9aa16dfca4f8)
For example, in the above referenced image, we can just have one job which can basically build, test and security-check for the iroha connector.
Also, having a global env variable to switch off all the security tests with in a go (if suppose we just want to test without the security context) would be good to have

Reply:
> @petermetz our CI will be flooded with lots of steps with 3 basic actions performed per plugin in general: building, testing, security-check. Shall we not club together all of these within `per-plugin` job, which does all the tasks per plugin together?
> 
> ![image](https://private-user-images.githubusercontent.com/56861721/282642358-39cb2609-aacf-48fc-8c46-9aa16dfca4f8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0MDEyMjQsIm5iZiI6MTcwMTQwMDkyNCwicGF0aCI6Ii81Njg2MTcyMS8yODI2NDIzNTgtMzljYjI2MDktYWFjZi00OGZjLThjNDYtOWFhMTZkZmNhNGY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAxVDAzMjIwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3ZTNjZjdmNTNjODZjZjM0ZjRiNWM5NGEyYWUzNTZjNjE3MGQ4NDUxMDgyNzBkNjExNzRiN2IyMGE4OTIxMjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vDuT7TsfGlO5e0xjOOaWi_aAwElj_67oVgJvj6_GSPw) For example, in the above referenced image, we can just have one job which can basically build, test and security-check for the iroha connector. Also, having a global env variable to switch off all the security tests with in a go (if suppose we just want to test without the security context) would be good to have

@jagpreetsinghsasan Agreed, 100%! Instead of new jobs, we can just add a new step within the job.
cc: @zondervancalvez

Reply:
addressed

Reply:
No, it isn't still addressed @zondervancalvez . Please have security checks as a part of already existing jobs (as a step). Also having a global var to turn off all the security scans in a single go is very helpful.